src/collective/sidebar/browser/sidebar.py

```diff
@@ -455,19 +456,14 @@ def get_action_icon(id1):
     """
     Returns icons for action ids
     """
-    icon_map = {
-        'cut': 'glyphicon glyphicon-scissors',
-        'copy': 'glyphicon glyphicon-duplicate',
-        'paste': 'glyphicon glyphicon-open-file',
-        'delete': 'glyphicon glyphicon-trash',
-        'rename': 'glyphicon glyphicon-random',
-        'ical_import_enable': 'glyphicon glyphicon-calendar',
-        'ical_import_disable': 'glyphicon glyphicon-calendar',
-    }
-    if id1 and id1 in icon_map:
-        return icon_map[id1]
+    icon_list = (
+        'cut', 'copy', 'paste', 'delete', 'rename', 'ical_import_enable',
+        'ical_import_disable',)
+
+    if id1 and id1 in icon_list:
+        return get_icon(id1)
     else:
```

Comment:
This `else` statement can be removed, because the condition always returns something.

Reply:
It returns a star-icon when there was no icon set for an object_action. (If you add your own action in the ZMI next to cut copy paste)